docs/nuget-org/nuget-org-faq.yml

```diff
@@ -60,33 +60,55 @@ sections:
           It is possible to share dependencies between multiple packages, making the total download size for consumers of your NuGet packages smaller.
           
           Dependencies are mostly static and never change. When fixing a bug in code, the dependencies may not need to be updated. If you bundle dependencies, you end up reshipping larger packages every time. By splitting NuGet packages into related dependencies, upgrades are much more fine-grained for consumers of your package.
-          
+
   - name: NuGet.org not accessible
     questions:
-      - question: Why can't I download packages from or upload packages to NuGet.org?
+      - question: What TLS version and cipher suites does NuGet.org support?
```

Comment:
```suggestion
      - question: What SSL/TLS version and cipher suites does NuGet.org support?
```

Reply:
Probably should just keep TLS given SSL is deprecated?

Reply:
I think the heading can shouldSSL but we mention SSL 1.0, 2.0, 3.0, TLS 1.0, and TLS 1.1 are disabled. Generally folks refer to TLS connections as SSL as a generalization. Include the term SSL might help Ctrl + F or Google searches.